packages/eas-cli/src/vcs/vcs.ts

```diff
@@ -58,4 +58,11 @@ export abstract class Client {
   public async getLastCommitMessageAsync(): Promise<string | null> {
     return null;
   }
+
+  // (optional) checks if the file is ignored
+  // - returns true if the file is included in the project tarball
+  // - returns false otherwise
```

Comment:
```suggestion
  // (optional) checks if the file is ignored, an implementation should ensure
  // that if file exists and `isFileIgnoredAsync` returns true, then that file 
  // should not be included in the project tarball.
```
it's not exactly correct
- true/false is inverted
- `if the file is included in the project tarball if it exists`